### ISSUE TITLE
Add release attachments to Pull Requests

### ISSUE THREAD

**Issue by FokkeZB** [isaacs/github#359]
While maintaining https://github.com/fokkezb/titanium-docs I ran into the problem that I'd like contributors to include a new distribution build when they do a PR.

**Comment by cirosantilli** [comment]
I don't understand the request clearly, can you be more precise? What is a "distribution build" ? How it is built? What are release attachments? Do you mean the files attached to repository tags?

**Comment by FokkeZB** [comment]
Indeed, attachments for releases like https://github.com/cirosantilli/test/releases/tag/a

The repo I referred to is a Desktop App for Windows, Mac and Linux. Contributors fork it, clone it, update it with the latest documentation and then build it for their own OS. They can do a Pull Request for the updated documentation, but have no way to include the build for their OS.

**Comment by cirosantilli** [comment]
Ah, OK, so you want it that when the contributor makes a pull request, they can also attach a file, and point somehow to an existing release, and if it gets merged the file is added to the list of files for the release? Or do you want them to actually create releases themselves if the PR gets accepted?

Or do you just want arbitrary non-image attachments to issues?

**Comment by FokkeZB** [comment]
I think ideally they would do a PR for a commit that includes a tag and has binaries attached to it. Then when I merge the PR I can turn the tag into a release with the same binaries.
